### DISCUSSION TITLE
Description of the Mirror mode for osu!catch is misleading

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19076] [Q&A] [answered]
Actually, the mode flips fruits **vertically**, not horizontally.

https://user-images.githubusercontent.com/86934170/178363105-e06e6820-9514-4f62-8a88-f5c301d71913.mp4

**Comment by plaguemask** [comment, marked as answer]
That's not how the vast majority of users understand flipping. Open basically any image editing software and it will disagree with you. The fruit are flipped around the vertical *axis*, but that is not how flipping is normally described. The flip made the fruit move horizontally, therefore it is a horizontal flip.

**Reply by Loreos7** [reply]
I got it now, thanks
